### PR TITLE
 feat(config): use table printer to format help message

### PR DESCRIPTION
**Pull request by kdada**
**What this PR does / why we need it**:

Use table printer to format help message.
```
➜  testnirvana ./testnirvana --help
ConfigKey-ENV-Flag Mapping Table

   Config Key                   ENV                          Flag                    Current Value
1  server.ip                    SERVER_IP                    --ip
2  server.metrics.namespace     SERVER_METRICS_NAMESPACE     --metrics-namespace
3  server.metrics.path          SERVER_METRICS_PATH          --metrics-path          /metrics
4  server.port                  SERVER_PORT                  --port                  8080
5  server.profiling.contention  SERVER_PROFILING_CONTENTION  --profiling-contention  false
6  server.profiling.path        SERVER_PROFILING_PATH        --profiling-path        /profiling

Usage:
  server [flags]

Flags:
  -h, --help                       help for server
      --ip string                  Nirvana server listening IP
      --metrics-namespace string   Metrics namespace
      --metrics-path string        Metrics path (default "/metrics")
      --port uint16                Nirvana server listening Port (default 8080)
      --profiling-contention       Enable contention profiling
      --profiling-path string      Profiling path (default "/profiling")
```

It's more readable than before.

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @zoumo 
/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: utils/printer/table_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package printer
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestTablePrinter(t *testing.T) {
+	lines := [][]interface{}{
+		{"Index", "Programing Language", "Percentage (Change)", "URL"},
```
Comment (ddysher) [inline utils/printer/table_test.go]: does this cover width too long?

Comment (kdada) [reply utils/printer/table_test.go]: No. I may add more tests in the future.

**Pull request merged**